lib/models/rloi.js

```diff
@@ -38,6 +40,7 @@ class Rloi {
               station = await this.s3.getObject({ Bucket: bucket, Key: `rloi/${item.$.region}/${item.$.stationReference}/station.json` })
               station = JSON.parse(station.Body)
             } catch (err) {
+              console.log({ err })
```

Comment:
For context this error log was left out original code so as not to spam the cloudwatch lambda logging, as the s3.getObject throws an error when it can't find the object, and there are a significant number of telemetry objects that we don't have a matching station for, and also hence the need to catch the error here.

Reply:
OK, it was useful while debugging but I'll add a comment and remove it.